src/media/local-video-track.ts

```diff
@@ -0,0 +1,130 @@
+import { logger } from '../util/logger';
+import { LocalTrack } from './local-track';
+
+export enum FacingMode {
+  user = 'user',
+  environment = 'environment',
+}
```

Comment:
There are built-in types for all these, no?  Can we leverage those?

Reply:
currently only MediaTrackCapabilities is exposed and not the sub types 
<img width="517" alt="Screenshot 2022-12-14 at 11 41 26 AM" src="https://user-images.githubusercontent.com/3895486/207655284-eb57e967-47db-4bb5-99ae-30e6a5b442ea.png">